src/LiveComponent/composer.json

```
@@ -56,7 +56,9 @@
         "zenstruck/foundry": "^2.0"
     },
     "conflict": {
-        "symfony/config": "<5.4.0"
+        "symfony/config": "<5.4.0",
+        "symfony/type-info": "<7.3",
+        "symfony/property-info": "~7.0"
```

Otherwise we can't install `symfony/property-info:>=7.1`
```suggestion
        "symfony/property-info": "~7.0.0"
```

---

Fixed